packages/server/lib/util/ci_provider.js

```diff
@@ -418,6 +425,12 @@ const _providerCommitParams = function () {
       // remoteOrigin: ???
       defaultBranch: env.DRONE_REPO_BRANCH,
     },
+    githubActions: {
+      sha: env.GITHUB_SHA,
+      branch: env.GITHUB_REF,
+      defaultBranch: env.GITHUB_BASE_REF,
+      remoteBranch: env.GITHUB_HEAD_REF,
```

Comment:
Are these already in `branchName` format? Or will they be full refs like the docs say, like `refs/heads/branchName`?